enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,533 @@
+   `install-config.controlPlane.platform.baremetal`
+5. The installer creates a new `arbiter` MachineSet with a replica of 1 and
+   checks the control plane replicas is `2` or higher
+6. With the object `arbiterNode` in the install config, the installer creates
+   the machine object with the correct node labels and machine config resources
+   for ignition bootstrap files.
+7. The installer applies the taint to the arbiter MachineSet
+8. The user can make any alterations to the node machine type to use less
+   powerful machines.
+9. The user then begins the install via `openshift-install create cluster`
+
+#### During Install
+
+1. The CEO will watch for new masters and the arbiter role
+2. CEO will create the operand for the etcd deployments that have tolerations
+   for the arbiter
+3. Operators that have tolerations for the arbiter should be scheduled on the
+   node
+4. The install should proceed as normal
+
+### API Extensions
+
+The `installConfig` will include a `install-config.controlPlane.arbiterNode`
+bool flag to denote arbiter infra structure.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+At the time being there is no impact on Hypershift since this edge deployment
+will require running the control plane.
+
+#### Standalone Clusters
+
+This change is relevant to standalone deployments of OpenShift at the edge or
+datacenters. This enhancement specifically deals with this type of deployment.
+
+#### Single-node Deployments or MicroShift
+
+This change does not effect Single Node or MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+Currently there are some behavior unknowns, we will need to put out a POC to
+validate some of the desires in this proposal. In it's current version this
+proposal is not exhaustive but will be filled out as we implement these goals.
+
+We currently expect this feature to mainly be used by `baremetal` installs, or
+specialized hardware that is built to take advantage of this type of
+configuration. In the current design we make two paths for cloud and baremetal
+installs in the installer. However, the cloud install is primarily for testing,
+this might mean that we simplify the installer changes if we are the only ones
+using cloud installs, since we can simply alter the manifests in the pipeline
+with out needing to change the installer.
+
+### Risks and Mitigations
+
+The main risk in this enhancement is that because we are treating one of the
+master nodes in a 3 node cluster as an arbiter, we are explicitly evicting
+processes that would otherwise be a normal supported upstream configuration such
+as a compact cluster. We run the risk of new components being critical to HA not
+containing the proper tolerations for running on the arbiter node. One of the
+mitigations we can take against that is to make sure we are testing installs and
+updates.
+
+Another risk we run is customers using an arbiter node with improper disk speeds
```

Comment:
I think there's a much higher risk that the network latency is impacting etcd negatively than the disk speed. 

See also https://github.com/etcd-io/etcd/issues/14501

Reply:
Ah, you're right, updated and thanks for the link, added it :)